src/examples/forms/input-group/Disabled.tsx

```diff
@@ -17,7 +17,9 @@ const Example = () => (
         <Label>Plant name generator</Label>
         <InputGroup>
           <Input value="Sheepberry" disabled />
-          <Button disabled>Germinate</Button>
+          <Button disabled isNeutral>
+            Germinate
```

Comment:
With the visual change, the word “Germinate” no longer reads as the action. Let’s change the copy to “Generate”.

Reply:
(this should be true in all the examples)

Reply:
updated.